integration/testomitempty.graphql

```diff
@@ -1,3 +1,25 @@
 type RemoteModelWithOmitempty {
     newDesc: String
 }
+
+type DummyUserWithRemoteNamedBasics {
```

Comment:
Can you change references like this from "RemoteNamedBasics" to DefinedTypeFromBasics?

Reply:
I updated the code and removed "DummyUserWith", just kept "DefinedTypeFromBasics".